procrastinate/postgres.py

```diff
@@ -31,6 +31,21 @@
   AND (%(task_name)s IS NULL OR task_name = %(task_name)s)
 """
 
+delete_old_jobs_sql = """
+DELETE FROM procrastinate_jobs
+WHERE status IN %(statuses)s
+  AND (%(queue)s IS NULL OR queue_name = %(queue)s)
+  AND (id IN (
+      SELECT DISTINCT ON (job_id) job_id
```

Comment:
This could be written with a JOIN, no ?

Reply:
I don't know how to do it, I can look into it :)